support-models/src/main/scala/com/gu/support/zuora/api/response/Responses.scala

```diff
@@ -115,7 +115,7 @@ case class GetPaymentMethodDirectDebitResponse(
 case class GetPaymentMethodCardReferenceResponse(
   `type`: String,
   paymentMethodStatus: String,
-  creditCardType: String,
+  creditCardType: Option[String],
```

Comment:
Payment methods can be reused fine when this is `None`?

Reply:
Correct. It's not needed in Zuora to process the payment, we just have it there for good measure.